### DISCUSSION TITLE
FreeBSD Setup?

### DISCUSSION THREAD

**Discussion by rblancarte** [GerryDazoo/Slinger#183] [Q&A]
Has anyone set this up on FreeBSD?

The scripts run fine, that's not the hard part.

What I'm looking for is help writing rc.d scripts for launching this on system start. Anyone?

**Comment by anonyada** [comment]
See section 4 for an explanation of BSD startup files and locations.
https://docs.freebsd.org/en/articles/linux-users/

**Reply by rblancarte** [reply]
The location isn't the issue. it's writing the file. I tried crafting 1 but it didn't execute right. It kept throwing errors about not being able to run my program.

I'm already on to this page
https://docs.freebsd.org/en/articles/rc-scripting/
